low-level/sensors/impl/generic/node_sas_port.py

```diff
@@ -76,7 +77,8 @@ class SASPortSensor(SensorThread, InternalMsgQ):
     POLLING_INTERVAL = "polling_interval"
     CACHE_DIR_NAME  = "server"
 
-    RESOURCE_ID = "SASHBA-0"
+    HOST_ID = SAS().get_host_list()[0].replace('host', '')
+    RESOURCE_ID = SAS_RESOURCE_ID + HOST_ID  # eg. SASHBA-0 if host_id=0
```

Comment:
Hi @cdeshmukh , Currently, in `SASPortSensor`, I have grabbed the host_id from sysfs, and simply aligned the resource id / uid for phys and ports according to our schema. 
I tried using port and phy info collected via new code in `SAS` class methods but it makes us changes nearly the whole Sensor code, increasing the scope of this PR. 
Is this approach okay?

Reply:
I understand the code changes here, which am fine with, its inline with our discussion

btw see if any change needed in existing sanity test, if any, for any of the asserted value

Reply:
Hi, I don't see any sanity test for SAS, we don't have mock data for SAS as well for tests on VM.

Reply:
then no changes needed for this PR at least. 
Need to plan to SAS tests in future though